www-servers/lighttpd/lighttpd-1.4.53.recipe

```diff
@@ -20,7 +20,7 @@ ADDITIONAL_FILES="
 	quickstart-geoip.txt
 	"
 
-ARCHITECTURES="all !x86_gcc2 ?arm ?ppc"
```

Comment:
did you test this on power?

Reply:
An unsupported platform? No.

Reply:
"all !x86_gcc2" means *every* arch defined except for x86_gcc2, I'm asking for the reason you marked arm and powerpc as tested if you haven't tested them at all.

Wether repos exist is irrelevant to wether recipes could be build for those arches.

Reply:
- Lighttpd is known to work/verified on BSD/Linux-ppc/ppc64/arm platforms. 

Why document on unsupported variants or a port known to work on certain processors or supported cross platform OSes similar to Haiku? We are discussing POWER in which there is no arch support yet for Haiku. Even for ppc, there are no image releases at all (just the CI kernel builds).  As for arm, lighttpd packages exist for BSD/Linux so why create more effort documenting a "maybe/maybe not" condition with "supported" ways to build/verify on Haiku-arm?

Reply:
The majority of the recipes at HaikuPorts make no mention of the other arches.  Obviously they haven't been tested either but I don't think we should go through and add the untested flag to all of them.  I seem to recall having a conversation with someone(maybe @waddlesplash) who said that we assume it works until we hear otherwise.

Reply:
Yes. Unless the recipe has some kind of per-architecture status (e.g. GCC) then `all` should always be used, with specific exceptions (likely `!x86_gcc2`) listed, not each individual architecture listed.

Reply:
So that is to say, the change as it currently exists is correct.